pages/price-feeds/use-real-time-data/evm.mdx

```diff
@@ -27,41 +24,22 @@ npm install @pythnetwork/pyth-sdk-solidity
 If you are using Foundry, you will need to create an NPM project if you don't already have one.
 From the root directory of your project, run:
 
-```bash
+```bash copy
 npm init -y
 npm install @pythnetwork/pyth-sdk-solidity
 ```
 
 Then add the following line to your `remappings.txt` file:
 
-```text
+```text copy
 @pythnetwork/pyth-sdk-solidity/=node_modules/@pythnetwork/pyth-sdk-solidity
 ```
```

Comment:
I think the typescript SDK is important -- it gets used below

Reply:
The guide title is to use Pyth in EVM Contracts. So, let's focus on that only. FE code will be moved to off-chain application. What do you think?